BoxContentSDK/BoxContentSDK/Operations/BOXAPIOperation.m

```diff
@@ -555,7 +555,17 @@ - (void)finishURLSessionTaskWithData:(NSData *)data response:(NSURLResponse *)re
 
 - (void)sessionTask:(NSURLSessionTask *)sessionTask didFinishWithResponse:(NSURLResponse *)response error:(NSError *)error
 {
-    [self finishURLSessionTaskWithData:nil response:response error:error];
+    [self finishURLSessionTaskWithData:self.responseData response:response error:error];
+}
+
+- (void)processIntermediateResponse:(NSURLResponse *)response
```

Comment:
Sometimes you need the response first... We might need to solve this another way